modules/p2/ding_serendipity_lists/ding_serendipity_lists.module

```diff
@@ -370,24 +377,26 @@ function ding_serendipity_lists_author_from_lists_serendipity_add($context) {
       // Fallback to dc.subject.
       if ($value === FALSE) {
         $value = 'facet.subject="' . $term->name . '"';
+        $field_filters[] = new TingSearchFieldFilter(TingSearchCommonFields::SUBJECT, $term->name, TingSearchFieldFilter::OP_AND);
```

Comment:
Do we need `$value` above now that we have `$field_filters`?

Reply:
nope